apps/site/util/blogUtils.ts

```diff
@@ -15,18 +12,3 @@ export const mapBlogCategoryToPreviewType = (type: string): BlogPreviewType => {
       return 'announcements';
   }
 };
-
-// @todo: we should check about the future of GitHub avatars
-// and mapping them to the respective users
-// @see https://github.com/nodejs/nodejs.dev/blob/main/src/data/blog/authors.yaml
-export const mapAuthorToCardAuthors = (author: string) => {
-  const authors = author.split(/, | and |;| & | prepared by | by /i);
```

Comment:
Is the removal of this safe? This splitting was intentional as some author strings within the markdown sources are well, complex, like in these examples.

Reply:
https://github.com/nodejs/nodejs.org/blob/1d1587d6697a015f30e26db055f616b13e3f266e/apps/site/util/authorUtils.ts#L6-L7

Reply:
Thanks, @AugustinMauroy! I was a bit busy yesterday so I couldn't reply 🙇